loopy/kernel/creation.py

```diff
@@ -2017,10 +2017,6 @@ def make_kernel(domains, instructions, kernel_data=["..."], **kwargs):
 
     if lang_version not in version_to_symbol:
         raise LoopyError("Language version '%s' is not known." % (lang_version,))
-    if lang_version >= (2018, 1):
-        options = options.copy(enforce_variable_access_ordered=True)
-    if lang_version >= (2018, 2):
-        options = options.copy(ignore_boostable_into=True)
```

Comment:
This should error out below the minimum version.

Reply:
Logic for this is already present
```
In [1]: import loopy as lp                                 

In [2]: knl = lp.make_kernel( 
   ...:     "{[i]: 0<=i<10}", 
   ...:     """ 
   ...:     y[i] = sin(x[i]) 
   ...:     """, lang_version=(2018, 1))                   
---------------------------------------------------------------------------
LoopyError                                Traceback (most recent call last)
<ipython-input-2-57402000f224> in <module>
----> 1 knl = lp.make_kernel(
      2     "{[i]: 0<=i<10}",
      3     """
      4     y[i] = sin(x[i])
      5     """, lang_version=(2018, 1))

~/pack/virtual-envs/loopy-dev/src/loopy/loopy/kernel/creation.py in make_kernel(domains, instructions, kernel_data, **kwargs)
   2017 
   2018     if lang_version not in version_to_symbol:
-> 2019         raise LoopyError("Language version '%s' is not known." % (lang_version,))
   2020 
   2021     # }}}

LoopyError: Language version '(2018, 1)' is not known.

```

Reply:
Thanks!